### DISCUSSION TITLE
Using MediaElement always seeks to position 0

### DISCUSSION THREAD

**Discussion by neilathomson** [katspaugh/wavesurfer.js#3242] [Q&A]
## Bug description
Using `MediaElement` allows playing before the whole audio is downloaded, however it breaks seeking (always seeks to position 0) even after the whole file is downloaded and the waveform renders.

## Environment
 - Browser: Chrome, Edge

## Minimal code snippet
```
const waveSurfer = WaveSurfer.create({
        container: document.body,
        barWidth: 3,
        cursorWidth: 1,
        backend: 'MediaElement',
        mediaControls: false,
        height: 70,
        responsive: true,
        splitChannels: false,
        waveColor: '#ff0000',
        progressColor: '#00ff00',
        cursorColor: '#0000ff',
        backgroundColor: '#ffffff',
      });

      waveSurfer.load("/api/recording");
```

## Expected result
Click somewhere on the waveform I expect it to move to that point.

## Obtained result
Click somewhere on the waveform it presently goes back to the start.

**Comment by katspaugh** [comment]
This is likely a bug with your audio file/server API, not wavesurfer.
